apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -40,6 +40,42 @@
 // Always include these helper files for dotcom FSE.
 require_once __DIR__ . '/dotcom-fse/helpers.php';
 
+/**
+ * Load Core Site Editor.
+ */
+function load_core_site_editor() {
+	$options = get_option( 'gutenberg-experiments' );
```

Comment:
`get_option()` accepts a second parameter that gets returned as a fallback for when the option is not found.

Reply:
Good to know!

The first time I ran this specific `get_option` on my sandbox it returned `false`, later after doing `update_option( 'slug', false );` it started returning nothing instead.  While that fallback would work for the nothing found case, I'm guessing it wouldn't work in the case where it returns `false` instead?

Reply:
True, in that case it doesn't help :)